node/task.ts

```diff
@@ -14,6 +14,7 @@ require('./extensions');
 
 export enum TaskResult {
     Succeeded = 0,
+    SucceededWithIssues = 2,
```

Comment:
`node/mock-task.ts` also needs the same change.

i dont see anywhere that parses a number back to the enum. the value is serialized using the enum name in `##` command. i'd rather change the order and keep this clean (succeeded=0, succeededwithissues=1, failed=2)